### ISSUE TITLE
Execution of a NeuralFMU with no parameters fails

### ISSUE THREAD

**Issue by TheGadgeteer** [ThummeTo/FMIFlux.jl#78]
If the chain only contains the fmu layer, the execution fails:
```julia
chain = FMIFlux.Chain(x -> fmu(;x=x)[2] )
neuralFMU = ME_NeuralFMU(fmu, chain, (0.0, 1.0), Tsit5())
solution = neuralFMU(x) # Throws BoundsError: attempt to access 0-element Vector{Bool} at index [1] 
```
whereas with a  layer with parameters, e.g.
```julia 
chain = FMIFlux.Chain(x -> fmu(;x=x)[2], x->Dense(2 => 2) )
```
 the code works.
The error is caused in neural.jl in line 1089:  ``` p_len = length(p[1]) ```   as p is empty.

**Comment by ThummeTo** [comment]
will be fixed in release v0.10.0

**Comment by ThummeTo** [comment]
closed with #79
